Cargo.toml

```diff
@@ -227,7 +229,6 @@ strip = "none"
 debug = true
 
 [profile.release]
-strip = "symbols"
```

Comment:
Is this change intentional?

Reply:
ah yeah, forgot to mention it. We're stripping symbols today which makes stack trace pretty much useless today. Out of curiosity I ran benchmarks with & without, and with `abort`/`unwind` for panic and saw no difference. So this parameter is just us hurting ourselves for the standard release